CHANGES.md

```diff
@@ -42,6 +42,7 @@ Features
 * [#422](https://github.com/twall/jna/pull/422): Update libffi to v3.2.1.
 * [#425](https://github.com/twall/jna/pull/425): Add support for PPCLE [@tdaitx](https://github.com/tdaix).
 * [#428](https://github.com/twall/jna/pull/428): Added some Wincon.h related functions and definitions
+* [#431](https://github.com/twall/jna/pull/431): Added named pipe API support
```

Comment:
No need - done, amended and pushed - including updating the log message to match. BTW, why the **manual** update ? Please note that there is a "chicken-and-egg" problem - a pull request number cannot be obtained until it is posted, and the _CHANGES.md_ cannot be updated until a pull-request number is available. In other words, there will always be **2** commits (or a commit + amend): one for the initial code, and the other for the line in _CHANGES.md_.

Why so complex ? The changes log is simply a list of all the commits since last release. If you want to avoid listing "minor" commits (TBD what it means), then associate each "major" one (whatever its definition) with an issue number and use a log message convention - e.g. ```[ISSUE 1234] Foo, bar, baz```.